docs/en/reference/working-with-associations.rst

```diff
@@ -32,62 +32,64 @@ information about its type and if it's the owning or inverse side.
 .. code-block:: php
 
     <?php
-    /** @Entity */
+    #[Entity]
     class User
     {
-        /** @Id @GeneratedValue @Column(type="string") */
-        private $id;
+        #[Id, GeneratedValue, Column(type: 'integer')]
+        private int|null $id;
```

Comment:
```suggestion
        private int|null $id = null;
```

We should initialize nullable properties with null to avoid the uninitialized state.